src/bvals/comms/bnd_info.hpp

```diff
@@ -127,11 +128,11 @@ struct ProResInfo {
 int GetBufferSize(MeshBlock *pmb, const NeighborBlock &nb,
                   std::shared_ptr<Variable<Real>> v);
 
-using BndInfoArr_t = ParArray1D<BndInfo>;
+using BndInfoArr_t = Kokkos::View<BndInfo *, LayoutWrapper, DevMemSpace>;
```

Comment:
No objection... just for my own understanding, can you explain the necessity of this change?

Reply:
We need to pass the `Kokkos::SequentialHostInit` property, which is done by not passing a string label but a properties object constructed via `Kokkos::view_alloc(Kokkos::SequentialHostInit, label)`.
This works for `Views` but `ParArray#D` are not plain `View` any more but carry a `state`.
Thus, in order to pass the properties object, we'd have to mirror that interface for `ParArray#D`, too.
Given that there's not place in the code (I could find) where we use the `state` for the outer view in a view of view, I decided to just use plain Views again (rather than extending the interface).

In order to not leak (more) "`Kokkos`"  pieces in the codebase, I'd be happy to introduce `RawParArray#D` that directly map to views if that gets a :+1: by more people.